sdk/src/androidTest/java/com/mapbox/maps/ViewAnnotationTest.kt

```diff
@@ -961,7 +961,7 @@ class ViewAnnotationTest(
   }
 
   private companion object {
-    const val VIEW_PLACEMENT_DELAY_MS = 1000L
+    const val VIEW_PLACEMENT_DELAY_MS = 3_000L
```

Comment:
Why?

Reply:
perhaps 1 sec is not enough to load a style and position view annotations on CI